tests/test_app_events.py

```diff
@@ -126,99 +125,8 @@ def test_stored_vocab_lookup_file_created_on_startup(
 ):
     """Test that on startup, a non-empty temporary lookup file is created for term ID-label mappings for the locally stored SNOMED CT vocabulary."""
     with test_app:
-        term_labels_path = test_app.app.state.vocab_lookup_paths["snomed"]
+        term_labels_path = test_app.app.state.vocab_lookup_paths[
+            "snomed_disorder"
```

Comment:
🍒: since we now create two SNOMED lookup files on startup instead of one, keyed on `snomed_disorder` and `snomed_assessment`, how about parametrizing this test to check that both paths exist?